tasks/section5.yml

```diff
@@ -539,7 +711,7 @@
       - rule_5.3.3
 
 - name: "SCORED | 5.3.4 | AUDIT | Ensure password hashing algorithm is SHA-512"
-  command: /bin/true
+  shell: authconfig --test | grep hashing | grep sha512
   register: result
```

Comment:
Audit tasks should register verbose variable name that ends with `_audit`. As you are adding AUDIT checks please change `register: result` to `register: `*`descriptive_check_name_audit`*